core-web/libs/sdk/react/src/lib/components/DotcmsLayout/DotcmsLayout.tsx

```diff
@@ -38,11 +38,12 @@ export type DotcmsPageProps = {
  * @returns {JSX.Element} - A JSX element that represents the layout for a DotCMS page.
  */
 export function DotcmsLayout({ pageContext, config }: DotcmsPageProps): JSX.Element {
-    const isInsideEditor = useDotcmsEditor(config);
+    const { isInsideEditor } = useDotcmsEditor(config);
+    pageContext = { ...pageContext, pageAsset: pageContext.pageAsset };
```

Comment:
What's the diff between `pageAsset` and `pageContext`, aren't we duplicating the data here?

Reply:
Yes that line is unnecessary, removed!